vite.config.ts

```diff
@@ -22,15 +27,35 @@ export default defineConfig({
 						'**/*.css',
 						'**/*.test.{js,jsx,ts,tsx}',
 						'**/__*.*',
-						// This is for server-side utilities you want to colocate next to
-						// your routes without making an additional directory.
-						// If you need a route that includes "server" or "client" in the
-						// filename, use the escape brackets like: my-route.[server].tsx
+						// This is for server-side utilities you want to colocate
+						// next to your routes without making an additional
+						// directory. If you need a route that includes "server" or
+						// "client" in the filename, use the escape brackets like:
+						// my-route.[server].tsx
 						'**/*.server.*',
 						'**/*.client.*',
 					],
 				})
 			},
 		}),
+		process.env.SENTRY_AUTH_TOKEN ??
```

Comment:
I'd prefer to use a ternary here:

```ts
process.env.SENTRY_AUTH_TOKEN ? sentryVitePlugin({ /* config */ }) : null
```

Reply:
ternary seems harder to read in a big config, I'm gonna have to look where's the `:` to finally find all that is declaring is a disappointing null

Reply:
if I read correctly, this peace of code, when SENTRY_AUTH_TOKEN is defined, sentryVitePlugin won't be included. so yeah it seems like the contrary of what we want

Reply:
👍